src/models/Settings.js

```diff
@@ -308,9 +308,6 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       if (!options.baseUrl) {
         this.callGlobalError(new ConfigError(Okta.loc('error.required.baseUrl')));
       }
-      else if (!options.globalSuccessFn) {
```

Comment:
This is a fair point.  We'd need to validate `authParams.display === 'page'` here, since this performs the redirect flow.

I think the idea moving forward is "Hey, the `success` handler isn't required for some flows, but it might be for yours."

@nbarbettini - Should we revisit adding the `Logger` when we **don't** have these callback methods?

Reply:
I don't want to add too much logic to these checks. Today, the success handler is required for authN flows and not required for OIDC flows, but that could change in the future.

What we're saying with this change is: if you don't add a success handler, nothing will happen on success. If you want to do something on success, add a success handler. I think that is clear and makes sense.